app/src/main/java/org/oppia/app/topic/revisioncard/ReturnToTopicClickListener.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.app.topic.revisioncard
+
+interface ReturnToTopicClickListener {
+  fun onReturnToTopicClicked()
+}
```

Comment:
EOF is missing, add one blank line at the end.

Reply:
Done